app/templates/gulpfile.js

```diff
@@ -64,9 +64,18 @@ gulp.task('html', ['styles'], () => {
 <% } -%>
   return gulp.src('app/*.html')
     .pipe($.useref({searchPath: ['.tmp', 'app', '.']}))
-    .pipe($.if('*.js', $.uglify()))
+    .pipe($.if('*.js', $.uglify({compress: {drop_console: true}})))
     .pipe($.if('*.css', $.cssnano({safe: true, autoprefixer: false})))
-    .pipe($.if('*.html', $.htmlmin({collapseWhitespace: true})))
+    .pipe($.if('*.html', $.htmlmin({
+      collapseWhitespace: true,
+      minifyCSS: true,
+      minifyJS: {compress: {drop_console: true}},
+      processConditionalComments: true,
+      removeComments: true,
+      removeEmptyAttributes: true,
+      removeScriptTypeAttributes: true,
+      removeStyleLinkTypeAttributes: true
+    })))
```

Comment:
Sure, my suggestions. Would you like inline comments, or just here for clarification?

```javascript
// https://github.com/kangax/html-minifier#options-quick-reference
$.htmlmin({
  collapseWhitespace: true,
  // Minify CSS in style elements and style attributes
  minifyCSS: true,
  // Minify JavaScript in script elements and event attributes
  minifyJS: {compress: {drop_console: true}},
  // Process contents of conditional comments through minifier
  processConditionalComments: true,
  // Strip HTML comments
  removeComments: true,
  // Remove all attributes with whitespace-only values
  removeEmptyAttributes: true,
  // Remove type="text/javascript" from script tags
  removeScriptTypeAttributes: true,
  // Remove type="text/css" from style and link tags
  removeStyleLinkTypeAttributes: true
})
```